build-system/tasks/check-json-schemas.js

```diff
@@ -0,0 +1,67 @@
+'use strict';
+
+const fs = require('fs');
+const json5 = require('json5');
+const path = require('path');
+const {cyan, green, red} = require('kleur/colors');
+const {default: addFormats} = require('ajv-formats');
+const {default: Ajv} = require('ajv');
+const {log} = require('../common/logging');
+
+/**
+ * Fetches the content of a JSON/JSON5 file.
+ *
+ * @param {string} file repo root relative.
+ * @return {any}
+ */
+function getJsonFile(file) {
```

Comment:
Since this is returning file contents (and not a file path or a file handle), maybe use a more explicit function name like `getJsonContents()`?